src/services/types.ts

```diff
@@ -492,12 +494,18 @@ namespace ts {
         kind: string;
     }
 
+    export interface JSDocTagInfo {

```

Comment:
I'm not sure about the naming scheme in services, but I'd prefer a terser name, like JSDocTag.